lib/helpers.coffee

```diff
@@ -1,8 +1,35 @@
 SelectorLinter = require 'atom-selector-linter'
+CSON = require 'season'
+fs = require 'fs-plus'
 
 exports.getSelectorDeprecations = ->
   linter = new SelectorLinter(maxPerPackage: 50)
   linter.checkPackage(pkg) for pkg in atom.packages.getLoadedPackages()
+  
+  userKeymapPath = atom.keymaps.getUserKeymapPath()
+  
+  if fs.existsSync(userKeymapPath)
+    linter.checkKeymap(CSON.readFileSync(atom.keymaps.getUserKeymapPath()), {
```

Comment:
`CSON.readFileSync` can throw an error so it should probably be caught here unless `getSelectorDeprecations` is already wrapped in a `try`/`catch`.